docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -12,8 +12,160 @@ weight = -17
+
+**Transformations shipping in 7.0**
+
+- **Reduce**: Reduce many rows / data points to a single value
+- **Filter by name**: Filter fields by name or regex
+- **Filter by refId**: Filter by query letter
+- **Organize fields**: Reorder, rename and hide fields.
+- **Labels to fields**: Transform time series with labels into a table where labels get's converted to fields and the result is joined by time
+- **Join by field**: Join many result sets (series) together using for example the time field. Useful for transforming time series into a table with a shared time column and where each series get it's own column.
+- **Add field from calculation**: This is a powerful transformation that allows you perform many different types of math operations and add the result as a new field. Examples:
+  - Calculate the difference between two series or fields and add the result to a new field
+  - Multiply one field with another another and add the result to a new field
+
+## Field overrides
+
+With Grafana 7.0 we are introducing a new, unified data configuration system that powers a consistent UI for setting data options across visualizations as well as making all data display settings data driven and overridable. This new option architecture and UI will make all panels have a consistent set of options and behaviors for attributes like `unit`, `min`, `max`, `thresholds`, `links`, `decimals` or `value mappings`. Not only that but all these options will share a consistent UI for specifying override rules and is extensible for custom panel specific options.
+
+Up until now the overrides were available only for Graph and Table panel(via Column Styles), but with 7.0 they work consistently across all visualisation types and plugins.
+
+This feature enables even more powerful visualizations and fine grained control over how the data is displayed.
+
+## Panel inspect and export to CSV
+
+Another new feature of Grafana 7.0 is Panel inspect. Inspect allows you to view the raw data for any Grafana panel as well as export that data to a CSV file. With Panel inspect you will also be able to perform simple raw data transformations like join, view query stats or detailed execution data.
+
+{{< docs-imagebox img="/img/docs/v70/panel_edit_export_raw_data.png" max-width="1024px" caption="Panel Edit - Export raw data to CSV" >}}
+{{< docs-imagebox img="/img/docs/v70/panel_edit_query_inspector.png" max-width="1024px" caption="Panel Edit - Query Inspector" >}}
+{{< docs-imagebox img="/img/docs/v70/panel_edit_raw_json.png" max-width="1024px" caption="Panel Edit - View raw JSON" >}}
+{{< docs-imagebox img="/img/docs/v70/panel_edit_stats.png" max-width="1024px" caption="Panel Edit - Stats" >}}
```

Comment:
A bit too many full size screenshots, with browser tabs :) Makes the article hard to read (A a lot of scrolling required). We can replace these images with the inspect video I have recorded, but not uploaded to youtube yet (scheduled for next Monday).

Reply:
Removed most of these images for now.